test/e2e/network/kube_proxy.go

```diff
@@ -229,7 +231,9 @@ var _ = SIGDescribe("Network", func() {
 		// Run the closewait command in a subroutine so it keeps waiting during postFinTimeoutSeconds
 		// otherwise the pod is deleted and the connection is closed loosing the conntrack entry
 		go func() {
-			framework.RunHostCmdOrDie(fr.Namespace.Name, "e2e-net-client", cmd)
+			defer ginkgo.GinkgoRecover()
```

Comment:
@spiffxp @BenTheElder 
I think this was the real problem, from https://onsi.github.io/ginkgo/
```
However, if your test launches a goroutine that calls Fail (or, equivalently, invokes a failing Gomega assertion), there’s no way for Ginkgo to rescue the panic that Fail invokes. This will cause the test suite to panic and no subsequent tests will run. To get around this you must rescue the panic using GinkgoRecover. Here’s an example:

It("panics in a goroutine", func(done Done) {
    go func() {
        defer GinkgoRecover()

        Ω(doSomething()).Should(BeTrue())

        close(done)
    }()
})
Now, if doSomething() returns false, Gomega will call Fail which will panic but the deferred GinkgoRecover() will recover said panic and prevent the test suite from exploding.
```